core/src/main/java/com/sequenceiq/cloudbreak/service/image/ImageCatalogService.java

```diff
@@ -375,25 +385,32 @@ private String latestCloudbreakVersion(Iterable<CloudbreakVersion> cloudbreakVer
         return sortedCloudbreakVersions.lastKey().getVersion();
     }
 
-    private Collection<String> prefixMatchForCBVersion(String cbVersion, Collection<CloudbreakVersion> cloudbreakVersions) {
+    private PrefixMatchImages prefixMatchForCBVersion(String cbVersion, Collection<CloudbreakVersion> cloudbreakVersions) {
```

Comment:
PrefixMatchedImages ?

Reply:
It is an internal object to return the images for a specific CB version which is not part of the catalog. Like 2.7.0-dev.500 then it will return all the images with 2.7.0 prefix